docs/pages/features/color-mode.md

```diff
@@ -0,0 +1,166 @@
+# Color Mode
```

Comment:
I think we should mark this page as a WIP. I'm still considering how to best provide the color modes in the template. But for now, we can use the `useChakra` hook.

```ts
import { useChakra } from '@chakra-ui/vue-next'

const { colorMode } = useChakra()

// You can also use the `useColorMode` hook
import { useColorMode } from '@chakra-ui/vue-next'

const { colorMode, toggleColorMode } = useColorMode()

```

Reply:
Should we just put a WIP message at the top and remove what is currently inside the doc file? And suggest using the `useChakra` hook?